client/scripts/views/components/sidebar/index.ts

```diff
@@ -248,10 +248,11 @@ export const ExternalLinksModule: m.Component<{}, {}> = {
 const Sidebar: m.Component<{ hideQuickSwitcher?, useQuickSwitcher?: boolean }, {}> = {
   view: (vnode) => {
     const { useQuickSwitcher } = vnode.attrs;
+    const isCustom = app.isCustomDomain();
 
     return [
-      !app.isCustomDomain() && m(SidebarQuickSwitcher),
-      !useQuickSwitcher && app.chain && m('.Sidebar', [
+      !isCustom && m(SidebarQuickSwitcher),
+      !useQuickSwitcher && app.chain && m(`.Sidebar${isCustom ? '.custom-domain' : ''}`, [
```

Comment:
the logic here can be simplified to `isCustom && '.custom-domain'`

Reply:
no need for the whole ternary

Reply:
I think this doesn't actually work because is resolving to a boolean when !isCustom, like ".Sidebarfalse".

Reply:
Ok